backend/app/graphql/services.py

```diff
@@ -1,14 +1,28 @@
+from typing import TypedDict
+from ..services.interfaces.user_service import IUserService
+from ..services.interfaces.auth_service import IAuthService
+from ..services.interfaces.email_service import IEmailService
+from ..services.interfaces.onboarding_request_service import IOnboardingRequestService
+from ..services.interfaces.meal_request_service import IMealRequestService
+
 """
 Global services for GraphQL that will be initialized with
 live app loggers during __init__.py
 """
-services = {
-    # Add your service here like this:
-    # "meal_request_service": None,
+
+
+class ServicesObject(TypedDict):
```

Comment:
I'm very excited about this change! Now when you do services["user_serivce"]. there is auto complete!

Reply:
Nice one!!